ambry-commons/src/main/java/com.github.ambry.commons/BlobId.java

```diff
@@ -498,6 +500,21 @@ public static boolean isCrafted(String idStr) throws IOException {
     return new Pair<>(blobIdPreamble.accountId, blobIdPreamble.containerId);
   }
 
+  /**
+   * Fetches {@link BlobId} from the given string representation of BlobId
+   * @param blobIdStr string representation of BlobId
+   * @param clusterMap {@link ClusterMap} instance to use
+   * @return the {@link BlobId} thus generated
+   * @throws RestServiceException
+   */
+  public static BlobId getBlobIdFromString(String blobIdStr, ClusterMap clusterMap) throws RestServiceException {
```

Comment:
Make sure to clean up this method

Reply:
oops. missed to remove it. Have removed it now.